src/sentry/issues/ingest.py

```diff
@@ -56,6 +57,13 @@ def save_issue_occurrence(
     group_info = save_issue_from_occurrence(occurrence, event, release)
     if group_info:
         send_issue_occurrence_to_eventstream(event, occurrence, group_info)
+
+        if (
+            occurrence.type.type_id in get_group_types_by_category(GroupCategory.PERFORMANCE.value)
+            and options.get("performance.issues.send_to_issues_platform", False)
+            and event.project.get_option("sentry:performance_issue_send_to_issues_platform", False)
+        ):
+            return
```

Comment:
Feels like we've repeating this a fair bit, does it make sense to wrap it in a utility function somewhere?